src/window.rs

```diff
@@ -168,7 +171,44 @@ impl WindowWrapper {
             previous_dpis,
             transparency: 1.0,
             fullscreen: false,
+            cached_width: 0,
```

Comment:
Minor: Why not store width and height as a tuple as well?

Reply:
I don't have a strong opinion on it, but it might be a bit ambiguous whether width or height comes first in a tuple.

Reply:
standard practice is width, height

Reply:
Noted, I'll change that aswell then